frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/PreviewPane/PreviewPane.styled.tsx

```diff
@@ -15,7 +19,21 @@ export const PreviewPaneContainer = styled.div<{
       position: absolute;
     `}
 
-  ${({ isTransparent }) =>
-    isTransparent &&
-    `background-image: url("app/img/pattern_checkerboard.svg")`};
+  ${({ backgroundType }) =>
+    match(backgroundType)
+      .returnType<SerializedStyles | null>()
+      .with("checkerboard-light", () => {
+        const dataUri = getCheckerBoardDataUri("checkerboard-light");
```

Comment:
Extract this to a component instead of inlining it because, inlining it causing `prettier` to fight with `styleling` because it wraps the the line.